opentech/apply/dashboard/tests/test_views.py

```diff
@@ -1,6 +1,6 @@
 from opentech.apply.funds.tests.factories import ApplicationSubmissionFactory, ApplicationRevisionFactory
 from opentech.apply.users.tests.factories import UserFactory, StaffFactory
-from opentech.apply.utils.tests import BaseViewTestCase
+from opentech.apply.utils.tests.tests import BaseViewTestCase
```

Comment:
?

Reply:
Ah i've got it. This is the test utils  rather than the tests for the utils. 

Maybe it should be `from opentech.apply.utils.testing import BaseViewTestCase` ? But not happy with that.

regardless, this naming isnt working